modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -0,0 +1,199 @@
+= Server Preparation for Ubuntu 18.04
+:hash-installation: http://php.net/manual/en/hash.installation.php
+:mcrypt-link-url: https://websiteforstudents.com/install-php-7-2-mcrypt-module-on-ubuntu-18-04-lts/
+:mcrypt-pecl-url: https://pecl.php.net/package/mcrypt
+:discover-samba-hosts: https://ubuntuforums.org/showthread.php?t=2384959
+:install-mariadb-latest: https://downloads.mariadb.org/mariadb/repositories/#
+
+== Webserver and php
+
+You only need to install __one__ of the following webservers.
+ 
+=== Apache Webserver + PHP (supported)
+
+The following command installs Apache webserver and php.
```

Comment:
Please change the sentence to "The following command installs the Apache web server and PHP."